src/main/scala/com/softwaremill/mqperf/mq/ArtemisMq.scala

```diff
@@ -0,0 +1,18 @@
+package com.softwaremill.mqperf.mq
+
+import javax.jms.ConnectionFactory
+
+import com.softwaremill.mqperf.config.TestConfig
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory
+
+class ArtemisMq(val testConfig: TestConfig) extends Jms2Mq {
+
+  override lazy val connectionFactory: ConnectionFactory = {
+    val hosts = "failover:(" + testConfig.brokerHosts.map(h => s"tcp://$h:61616").mkString(",") + ")"
```

Comment:
so here I'm getting:

```
java.lang.NullPointerException: Schema failover not found
	at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.<init>(ActiveMQConnectionFactory.java:212)
```

probably this needs to be configured differently in Artemis?